src/tools/pub/cmd/pub-uploader.md

```diff
@@ -18,17 +18,17 @@ $ pub uploader [options] {add/remove} <email>
 
 This command allows
 [uploaders](/tools/pub/glossary#uploader) of a
-package on the [pub.dev site]({{site.pub}}) to add or remove
+package on the [pub.dev site]({{site.pub}}) to add (invite) or remove
 other uploaders for that package. It has two sub-commands,
 `add` and `remove`, that take the email address of the person to
 add/remove as an uploader. For example:
 
 ```terminal
 ~/code/transmogrify$ pub uploader add bob@example.com
-'bob@example.com' added as an uploader for package 'transmogrify'.
+We have sent an invitation to bob@example.com, they will be added as uploader after they confirm it.
 
 ~/code/transmogrify$ pub uploader remove bob@example.com
-'bob@example.com' is no longer an uploader for package 'transmogrify'.
+Successfully removed uploader from package.
```

Comment:
I liked the previous text. Can we tweak this to include at least the email address? Something like this:

```suggestion
bob@example.com has been removed as an uploader for this package.
```